skycoin/skywire
# Release/v0.4.2

Did you run `make format && make check`?

Fixes #	

 Changes:	
- Vendors release dmsg
- Increase uptime tracker update interval
- Removed skysocks service discovery registration
- Update config with new url
How to test this PR:
1. Run skywire-visor
2. Check if config is updated to `1.0.1` and the URLs are updated.
3. Check if there is not other error
